sdk/identity/identity-spa/review/identity-spa.api.md

```diff
@@ -0,0 +1,45 @@
+## API Report File for "@azure/identity-spa"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AccessToken } from '@azure/core-auth';
+import { AuthenticationRecord } from '@azure/identity';
+import { CredentialPersistenceOptions } from '@azure/identity';
+import { InteractiveCredentialOptions } from '@azure/identity';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export class PopupCredential implements TokenCredential {
+    constructor(_options: SPACredentialOptions);
+    authenticate(): Promise<AuthenticationRecord | undefined>;
+    getToken(): Promise<AccessToken | null>;
+    onPageLoad(): Promise<{
+        code?: string;
+        state?: string;
+    }>;
+}
+
+// @public
+export class RedirectCredential implements TokenCredential {
+    constructor(_options: SPACredentialOptions);
+    authenticate(): Promise<AuthenticationRecord | undefined>;
+    getToken(): Promise<AccessToken | null>;
+    onPageLoad(): Promise<{
+        code?: string;
+        state?: string;
+    }>;
+}
+
+// @public
+export interface SPACredentialOptions extends InteractiveCredentialOptions, CredentialPersistenceOptions {
+    clientId?: string;
```

Comment:
CredentialPersistenceOptions not needed, since the persistence is already done by the MSAL browser